### ISSUE TITLE
External Youtube videos do not work

### ISSUE THREAD

**Issue by 54m4d** [TheFantasticWarrior/chrome-extension-imagus#25]
Hello,

I have observed that the yt sieve does not function with external YouTube videos, such as those found on the Google search page, it is important to note that this issue is not related to the sieve itself, but rather with the extension, i have tested the original extension, and external YouTube videos play without any problems, therefore, it seems that no troubleshooting is required in this case, also no SMH is required.

Has this behavior been modified in the mod? or is there another factor contributing to this issue?

**Comment by ongots** [comment]
https://www.reddit.com/r/imagus/comments/14fyk2l/youtube_sieve_just_stopped_working/

**Comment by 54m4d** [comment]
You need to read

**Comment by TheFantasticWarrior** [comment]
#21 I changed the xhr call from background script to the content script, which fixes cookie bugs on firefox(in incognito the cookie can be set/accessed by websites) but then the xhr call will be blocked by the browser CORS
maybe there are other ways to set the call as incognito in the background script, but I don't know how yet 

https://www.reddit.com/r/imagus/comments/13kn4nu/firefox_private_mode_allows_for_setting_cookies/jqpw23k/?context=3
https://www.reddit.com/r/imagus/comments/121oop7/comment/jottktw/?utm_source=share&utm_medium=web2x&context=3

**Comment by 54m4d** [comment]
Is it possible to allow the xhrs in Chrome, is it also affected?

**Comment by TheFantasticWarrior** [comment]
chrome doesn't have the bug, so I probably need to maintain 2 versions until I find another way, for now just use v0.10.11

**Comment by 54m4d** [comment]
Yeah, that's what i was thinking, but the difference is minor, maybe just add a Chrome branch, and the funny thing is that the name of this repo is Chrome extension 😁

**Comment by TheFantasticWarrior** [comment]
I separated the versions in v0.10.14, should work now

**Comment by 54m4d** [comment]
Thanks for your effort!

Possible to remove Gecko only features from Chromium? like `"Change file extensions (Firefox only?)"`

**Comment by TheFantasticWarrior** [comment]
I haven't done thorough testing, it seems that chrome defaults(forces) the file extension, but after more searching I found [this](https://apple.stackexchange.com/questions/444970/chrome-doesnt-always-add-file-extension-to-downloaded-images), on the first download call no file extension is used meaning the feature actually works on chrome too. I will edit the text in a future update

Also I removed most gecko only features, but probably missed some. I'll close this issue and if there are other features I missed make a new issue

**Comment by 54m4d** [comment]
OK, if what i mentioned above applies to Chrome too, then all OK.
